sdk/src/main/java/io/opentelemetry/sdk/trace/export/ExporterConstants.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace.export;
+
+import java.util.concurrent.TimeUnit;
+
+/** Provides constants for all exporter default configuration values. */
+public class ExporterConstants {
+
+  private ExporterConstants() {}
+
+  /** Common default host name. */
+  public static final String DEFAULT_HOST_NAME = "unknown";
```

Comment:
Why unknown default?

Reply:
The default value used now is unknown. see [https://github.com/open-telemetry/opentelemetry-java/blob/78ed6490ba47568f5241604ea34f18d3ae6f3c4a/exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java#L170](https://github.com/open-telemetry/opentelemetry-java/blob/78ed6490ba47568f5241604ea34f18d3ae6f3c4a/exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java#L170)
[https://github.com/open-telemetry/opentelemetry-java/blob/78ed6490ba47568f5241604ea34f18d3ae6f3c4a/exporters/zipkin/src/main/java/io/opentelemetry/exporters/zipkin/ZipkinSpanExporter.java#L282](https://github.com/open-telemetry/opentelemetry-java/blob/78ed6490ba47568f5241604ea34f18d3ae6f3c4a/exporters/zipkin/src/main/java/io/opentelemetry/exporters/zipkin/ZipkinSpanExporter.java#L282)